src/components/banner/template.test.tsx

```diff
@@ -27,23 +27,18 @@ describe('<Banner> component renders', () => {
   test('with dismiss permanently', () => {
     const mockBannerData = getMockBannerData()
     render(<Banner {...mockBannerData} dismiss={true} />)
-    expect(
-      screen.getByText(
-        /The banner component is part of the VSP Design System Storybook/
-      )
-    ).toBeInTheDocument()
+    expect(screen.getByTestId('banner')).toHaveAttribute('show-close')
```

Comment:
Locally during commit hook this renders `show-close = true` the server just renders `show-close` so these tests are a little cheesy at the moment

Reply:
but at least they are actually testing something now